Sources/PortalSwift/Swaps/PortalSwaps.swift

```diff
@@ -48,58 +48,58 @@ public struct QuoteArgs: Codable {
     self.sellAmount = sellAmount
```

Comment:
Updated to use the new `AnyEncodable` to conform to new `PortalRequests` class restrictions